pkg/lib/page.css

```diff
@@ -243,51 +243,325 @@ input[type=number] {
-    padding: 1px 28px 1px 5px;
-    border: 1px solid #bbb;
+    padding: 0.25em 2.5em calc(0.25em - 1px) 0.5em;
+    border: 1px solid var(--pf-global--BorderColor--300);
+    border-bottom-color: var(--pf-global--Color--200);
     background: var(--dropdown-image) no-repeat 100% 50%, var(--dropdown-background);
-    box-shadow: 0 2px 3px rgba(3, 3, 3, 0.1);
-    border-radius: 1px;
-    color: #4d5258;
+    border-radius: 3px;
+    color: var(--pf-global--Color--100);
     cursor: pointer;
     transition: border-color ease-in-out 0.15s, box-shadow ease-in-out 0.15s;
-    font-weight: 600;
-    min-height: 26px;
+    min-height: 2.25rem;
+    box-shadow: none;
 }
 
 @-moz-document url-prefix() {
     /* Accomodate Firefox styling selects with slightly different padding. */
     .ct-select {
-        padding-left: 1px;
+        padding-left: 0.25em;
     }
 }
 
-.ct-select:hover,
-.ct-select:active {
-    --dropdown-background: #f1f1f1;
-}
-
+.ct-select:active,
+.ct-select:focus,
 .ct-select:hover {
-    border-color: #7dc3e8;
+    border-bottom-color: var(--pf-global--primary-color--100);
+    box-shadow: none;
 }
 
+.ct-select:active,
 .ct-select:focus {
-    border-color: #0088ce; /* pf-blue-400 */
-    /* Outer blue glow */
-    box-shadow: inset 0 1px 1px rgba(3, 3, 3, 0.075), 0 0 8px rgba(0, 136, 206, 0.6);
-    color: #4d5258;
+    border-bottom-width: 2px;
+    padding-bottom: calc(0.25rem - 2px);
 }
 
-.ct-select:active {
-    box-shadow: inset 0 2px 8px rgba(3, 3, 3, 0.2);
+.ct-select:disabled {
+    --dropdown-background: var(--pf-global--disabled-color--300);
+    color: var(--pf-global--disabled-color--100);
+    border: 1px solid var(--dropdown-background);
+    cursor: default;
 }
 
 .ct-select optgroup,
 .ct-select option {
-    background-color: #fff;
+    background-color: var(--color-bg);
+}
+
+.ct-select option:checked {
+    background-color: var(--pf-global--active-color--100);
+    color: var(--pf-global--Color--light-100);
+}
+
+:root {
+    /* Custom font aliases */
+    --font-family: var(--pf-global--FontFamily--sans-serif);
+    --font-family-mono: var(--pf-global--FontFamily--monospace);
+    --font-size: var(--pf-global--FontSize--md);
+    --font-size-s: var(--pf-global--FontSize--sm);
+    --font-size-xs: var(--pf-global--FontSize--xs);
+    --font-size-l: var(--pf-global--FontSize--lg);
+    --font-size-xl: var(--pf-global--FontSize--xl);
+    --font-size-2xl: var(--pf-global--FontSize--2xl);
+    --font-size-3xl: var(--pf-global--FontSize--3xl);
+
+    /* Cockpit custom colors */
+    --color-ct-light-red-1: #f8cccc;
+    --color-ct-light-red-2: #f8acac;
+    --color-ct-light-red-3: #f98383;
+    --color-ct-brick-gold : #ffa763;
+    --color-ct-brick-red  : #a53131;
+
+    --color-ct-nav            : #292e34;
+    --color-ct-nav-sub        : var(--pf-global--BackgroundColor--100);
+    --color-ct-nav-text       : var(--pf-global--Color--100);
+    --color-ct-nav-active     : var(--color-light-blue-2);
```

Comment:
Why `var(--color-light-blue-2)` instead of `var(--pf-global--BackgroundColor--150);` here?
I compared it to the [example page](https://www.patternfly.org/v4/documentation/react/demos/pagelayout/pagelayoutsimplenav/) and that uses the same colors for both active and hover.

Reply:
I specifically deviate from PatternFly 4 for the navigation because it just simply looks wrong with the top-level navigation.

Reply:
We can experiment with alternate colors and update it though. I'm happy to work with you on figuring out a solution, if the light blue doesn't work.

Reply:
Could something like this work?
![Screenshot from 2019-06-19 15-41-28](https://user-images.githubusercontent.com/209895/59771415-f475cd00-92a9-11e9-8df2-baabd36ab313.png)